language/compiler/ir_to_bytecode/syntax/src/syntax.lalrpop

```diff
@@ -343,12 +344,45 @@ FunctionBlock: (Vec<(Var_, Type)>, Block) = {
     "{" <locals: Declarations> <stmts: Statements> "}" => (locals, Block::new(stmts))
 }
 
+KindConstraint: KindConstraint = {
```

Comment:
Nit I'd call this TypeParameter

Reply:
Yeah I noticed you changed the name in your diff. I'll make sure they are consistent before checking this in.